index.js

```diff
@@ -0,0 +1,124 @@
+const java_version = [
+  "openjdk8",
+  "openjdk9",
+  "openjdk10",
+  "openjdk11"
+]
+
+// This is the total number displayed on the page
+let total = 0;
+let counter = 0;
+for (var version of java_version) {
+  var settings = {
+    "url": `https://api.adoptopenjdk.net/v2/info/releases/${version}`,
+    "method": "GET"
+  }
+
+  $.ajax(settings).done(function(releases) {
+    counter += 1;
+    for (var release of releases) {
+      total += release.download_count
+    }
+    if (counter == java_version.length) {
```

Comment:
OK, can we split out some of the logic into named functions?  This seems to be the classic for if for nested JS of doom :-)

Reply:
done